src/Form.jsx

```diff
@@ -94,6 +92,8 @@ function Form({
     const [stepSequence, setStepSequence] = useState([]);
     const [sequenceIndex, setSequenceIndex] = useState(0);
 
+    const [firebase, setFirebase] = useState(null);
```

Comment:
Do we need to be careful about checking for null when we use `firebase`?

Reply:
imo we'd rather hard fail if firebase isn't there and we expect it to be there, since that's indicative of a more serious breakage in our upstream assumptions

Reply:
right now it'd be if we had a login field in a form that had sms authentication but the firebase integration is turned on. which shouldn't be allowed anyways